Lib/test/test_tix.py

```diff
@@ -24,9 +28,9 @@ def setUp(self):
         else:
             self.addCleanup(self.root.destroy)
 
-    def test_tix_available(self):
-        # this test is just here to make setUp run
-        pass
+    def test_tix_warning(self):
+        with self.assertWarns(DeprecationWarning):
+            from tkinter import tix
```

Comment:
You can try `support.import_fresh_module('tkinter.tix', fresh=('tkinter.tix',))` to import fresh with the warning.